include/deal.II/boost_adaptors/point.h

```diff
@@ -0,0 +1,92 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2018 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_boost_adaptor_point_h
+#define dealii_boost_adaptor_point_h
+
+#include <deal.II/base/point.h>
+
+#include <boost/geometry.hpp>
+
+
+namespace boost
+{
+  namespace geometry
+  {
+    namespace traits
+    {
+      /**
+       * tag adaptor for dealii::Point
+       */
+      template <class Number, size_t dim>
+      struct tag<dealii::Point<dim, Number>>
```

Comment:
I would have reverted the template arguments in the template header, so that they match the template argument list of class `Point`